docs/content/en/docs/environment/templating.md

```diff
@@ -23,6 +23,7 @@ List of fields that support templating:
 * `deploy.kubectl.defaultNamespace`
 * `deploy.kustomize.defaultNamespace`
 * `portForward.namespace`
+* `portForward.resourceName`
```

Comment:
nit: Should this be `resourceName` or `Name`?

Reply:
It is correct, as in code we have Name, but in schema https://github.com/GoogleContainerTools/skaffold/blob/aa9b115dbf1559312b317f8ad47117fd4d56a361/pkg/skaffold/schema/latest/v1/config.go#L132